lib/keyring.js

```diff
@@ -66,6 +67,53 @@ KeyRing.prototype._migrateOld = function(oldPath) {
 };
 
 /**
+* @param {String} keyring - path to keyring to be compressed
+* @param {String} tar - Path to compressed keyring
+ */
+KeyRing.prototype.export = function(tar, callback) {
+  var read = targz().createReadStream(this._keyFolder);
+  var write = fs.createWriteStream(tar);
```

Comment:
Can we check if the path already exists and if so callback with an error refusing to overwrite?